app/models/course/video.rb

```diff
@@ -40,6 +40,8 @@ class Course::Video < ApplicationRecord
                     pluck('DISTINCT video_id'))
   end)
 
+  scope :active, -> { joins(:lesson_plan_item).merge(Course::LessonPlan::Item.currently_active) }
```

Comment:
Would it be better if we shift this scope into the `acts_as_lesson_plan_item`  declaration for the rest of the other lesson plan items to use?

Reply:
Sure, I can shift it.